.github/workflows/firebase-hosting-merge.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: npm ci && npm run build
+      - run: npm ci && npm run lint && npm run build
```

Comment:
Adding it here because I really don't care for adding extra stuff in the npm build script.  I feel they are two separate tasks.